src/main/java/frc/robot/subsystems/arm/back/BHighPurple.java

```diff
@@ -34,7 +40,12 @@ public void init() {
 
     @Override
     public void execute() {
+        RobotMap.arm.moveArmPosition(firstStagePosition, secondStagePosition, wristPosition);
 
+        // if arm has arrived at position, send open request to claw
+        if (RobotMap.arm.getArrived(allowance)) {
```

Comment:
It seems like we are kicking off something here that should be a different robot state.